app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<h3>Job Count by Level of Interest</h3>
+
+<table>
+  <thead>
+    <% Dashboard.build_range.each do |range| %>
```

Comment:
should never make direct calls to a model from the view. we can avoid this by passing them as instance variables through the controller.